tracing-subscriber/src/layer/mod.rs

```diff
@@ -1054,6 +1054,21 @@ feature! {
         fn max_level_hint(&self) -> Option<LevelFilter> {
             None
         }
+
+        /// Notifies this filter that a new span was constructed with the given
+        /// `Attributes` and `Id`.
+        fn on_new_span(&self, attrs: &span::Attributes<'_>, id: &span::Id, ctx: Context<'_, S>) {
+            let _ = (attrs, id, ctx);
+        }
+
+        /// Notifies this filter that a span with the given ID was entered.
+        fn on_enter(&self, _id: &span::Id, _ctx: Context<'_, S>) {}
```

Comment:
Should I maybe remove the underscores from the args (and add the `let _ = (..args)` thing to the body)?

Reply:
gladly, I've encountered these exact issues :)